MK3_3.12: Add Menu Calibration -> Temp Model cal.

3d-gussner opened the pull request:
Runs M310 A F0
Enables with M310 S1
Saves settings M500

Tested on MK404 with Octoprint serial logging.

3d-gussner committed:
Temp Model cal. menu

3d-gussner committed:
Fix MK25/S builds

wavexx commented on Firmware/temp_model.h:
```diff
@@ -59,6 +59,7 @@ struct model_data
 };
 
 static bool enabled;          // model check enabled
+static bool valid = false;    // model is valid
```
Why is this needed? temp_model::calibrated() should do the same and avoid having to set/reset another state variable?

3d-gussner committed:
Fix typo
Add lcd_setatatus_serial to output lcd status message and serial at the same time
